### PR TITLE
LPS-161759 | Master

### PR DESCRIPTION
**Pull request by Jeremy-Ch**
https://issues.liferay.com/browse/LPS-161759

### CODE REVIEW

**Comment by Jeremy-Ch** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/knowledgebase/KBArticle.testcase [hunk 1]
```diff
@@ -731,6 +731,111 @@ definition {
...
+
+		CKEditor.addContent(content = "Knowledge Base Article Content");
+
+		KBArticle.setReviewDate(
+			enableReviewDate = "true",
+			increaseMinutes = 1);
+
+		PortletEntry.publish();
+
+		KBAdmin.openKBAdmin(siteURLKey = "guest");
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/collaboration/knowledgebase/KBArticle.testcase:756]: Please add as conditional to KBArticle.addCP. I asked Lily to do the same for expiration dates here https://github.com/liferay-lima/liferay-portal/pull/4017/commits/f5ab18ea135a95fa051aad378059b811ba0b68e2

**Review by austinchiang — CHANGES_REQUESTED** [review]
Please make the changes necessary and resend

**Comment by austinchiang** [comment]
ci:help

**Comment by Jeremy-Ch** [comment]
See the new pull request: https://github.com/liferay-lima/liferay-portal/pull/4039

**Pull request closed without merging**